okapi-core/src/main/java/org/folio/okapi/bean/InstallJob.java

```diff
@@ -0,0 +1,49 @@
+package org.folio.okapi.bean;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import java.util.List;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class InstallJob {
+
+  private String id;
+
+  private Boolean complete;
+
+  private String date;
```

Comment:
This is the starting date. Is it possible to add a finish date to know the duration of the install/upgrade?

Reply:
ok. done